primitives/npos-elections/src/assignments.rs

```diff
@@ -0,0 +1,201 @@
+// This file is part of Substrate.
+
+// Copyright (C) 2020-2021 Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! Structs and helpers for distributing a voter's stake among various winners.
+
+use crate::{Error, ExtendedBalance, IdentifierT, PerThing128, __OrInvalidIndex};
+use codec::{Encode, Decode};
+use sp_arithmetic::{traits::{Bounded, Zero}, Normalizable, PerThing};
+use sp_core::RuntimeDebug;
+use sp_std::vec::Vec;
+
+/// A voter's stake assignment among a set of targets, represented as ratios.
+#[derive(RuntimeDebug, Clone, Default)]
+#[cfg_attr(feature = "std", derive(PartialEq, Eq, Encode, Decode))]
+pub struct Assignment<AccountId, P: PerThing> {
```

Comment:
no chnages to these I presume?

Reply:
No, just moved them out into a separate module because IMO it's easier to navigate lots of small files than one massive one.